tests/framework-tests/core/js/DataBindingTests.js

```diff
@@ -911,6 +911,97 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         jqUnit.assertDeepEq("Captured model by argument", {x: 30, y: 30}, that.frozenModel.windowHolders.mainWindow);
     });
 
+    /** FLUID-6127: Wildcards in modelListeners, and support for deletion **/
+
+    fluid.defaults("fluid.tests.fluid6127root", {
+        gradeNames: ["fluid.modelComponent", "fluid.tests.changeRecorder"],
+        modelListeners: {
+            wildcardy: {
+                path: "idToPath.*",
+                excludeSource: "init",
+                listener: "{that}.record", // TODO: support compact form here
+                args: ["{change}.path", "{change}.value", "{change}.oldValue"]
+            }
+        },
+        model: {
+            idToPath: {
+                first: 1
+            }
+        }
+    });
+
+    fluid.tests.fluid6127fixtures = [{
+        change: {
+            type: "ADD",
```

Comment:
There is currently no other separate change type in the system - the default when the type is not specified is "ADD"